### PR TITLE
update-waits

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

**Comment by paweldomas** [comment]
thanks !

**Comment by paweldomas** [comment]
I will do few changes on top though

**Comment by damencho** [comment]
There is one commit I forgot to push.

**Comment by damencho** [comment]
I can make a separate PR for that.

File: src/test/java/org/jitsi/meet/test/ChatPanelTest.java [hunk 1]
```diff
@@ -76,27 +76,21 @@ public void testChatPanel()
...
-        TestUtils.waitMillis(1000);
-        assertTrue(
-            chatPanel.isOpen(),
-            "The chat panel should be open after clicking the button");
+
+        // The chat panel should be open after clicking the button
+        WebDriverWait wait = new WebDriverWait(participant.getDriver(), 2);
+        wait.until((ExpectedCondition<Boolean>) d -> chatPanel.isOpen());
 
         chatPanel.pressShortcut();
```
Comment (paweldomas) [inline src/test/java/org/jitsi/meet/test/ChatPanelTest.java]: do you know why we first click the toolbar button and then use the shortcut ? wouldn't that result in open and close ?

Comment (paweldomas) [reply src/test/java/org/jitsi/meet/test/ChatPanelTest.java]: oh I see it's used to close it...

**Comment by paweldomas** [comment]
@damencho please take a look on added commits and comment or merge

**Pull request merged**